kge/dataset.py

```diff
@@ -91,6 +93,53 @@ def load(config: Config, preload_data=True):
                 dataset.split(split)
         return dataset
 
+    @staticmethod
+    def create_from(
+        checkpoint: Dict,
+        config: Config,
+        dataset: Optional[Dataset] = None,
+        preload_data=True,
+    ) -> Dataset:
+        """
+        Creates dataset based on a checkpoint.
+        If a dataset is provided it will be updated
+        Args:
+            checkpoint: loaded checkpoint
+            config: config created from checkpoint
+            dataset: dataset to update
+            preload_data: preload data
+
+        Returns: created/updated dataset
+
+        """
+        if dataset is None:
+            dataset = Dataset.load(config, preload_data)
```

Comment:
ah, you are right, this does not work anymore. We would need some parameter to check if the created dataset is a mocked dataset, so that it throws an error, when we create a job.
Should I just add a parameter "mocked" to dataset or do you have a better idea, how I could handle that in job.create_from without just reloading the dataset?

Reply:
How was this resolved?

Reply:
if preload_data is set to False, it will only create the dataset object without loading anything